src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -157,30 +226,34 @@ If you see the `java.lang.NoClassDefFoundError` error, then you must [manually s
 
 1. Create a `proguard.multidex.config` file within the `/app` folder of your project. Update `mypackage` to reflect your package name.
 
-  ```
-  ####################
-  # keep class names #
-  ####################
-
-  #Keep New Relic in the main dex
-  -keep class com.newrelic.** { *; }
-  -keep class com.mypackage.activities.** { *; }
-  ```
-2. Merge the following code into the app-level `build.gradle` file:
+```
+####################
+# keep class names #
+####################
 
+#Keep New Relic in the main dex
+-keep class com.newrelic.** { *; }
+-keep class com.mypackage.activities.** { *; }
+```
 
-  ```
-  android { 
-      defaultConfig{
-          …
-          multiDexKeepProguard file("proguard.multidex.config")
-      }
-  }
-  ```
+2. Merge the following code into the app-level `build.gradle` file:
 
+```
+android {
+    defaultConfig{
+        …
+        multiDexKeepProguard file("proguard.multidex.config")
+    }
+}
+```
 
 For more information, see the [Android Developers documentation](https://developer.android.com/studio/build/multidex.html#keep) on declaring classes required in the primary DEX file.
 
-## New Relic Gradle Plugin configuration [#gradle_plugin_config]
+## What's next? 
 
-For details on how to configure the behavior of the mobile monitoring agent plugin during Gradle builds, see the [New Relic Gradle plugin](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-new-relic-gradle-plugin) extension guide.
+Congratulations! You've successfully installed the mobile monitoring agent to your Android app. You have a few options for next steps:
+
+* Set up ProGuard or DexGuard BANANA with our [Configure ProGuard or DexGuard for Android apps](/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps) doc.
```

Comment:
Note to self: check if the step that directs users to update a root file is in the proguard doc. if it isn't, then add it